packages/apps/spaces/pages/api/auth/[...nextauth].ts

```diff
@@ -39,6 +39,22 @@ export const authOptions: AuthOptions = {
         token.id = (profile as GoogleProfile)?.id;
         token.playerIdentityId = account.providerAccountId;
 
+        // Check if the email is available in the profile
+        if (profile && profile.email && account.provider === 'azure-ad') {
```

Comment:
shouldn't be google here and azure on else?

Reply:
can you check again?